tox.ini

```diff
@@ -28,6 +28,10 @@ commands = flake8
 commands = {posargs}
 passenv = *
 
```

Comment:
Stuff like the _init_config I kinda just took for granted as just working. Looking a bit at it now, Id like to ask why we look at the virtualenv name?

Reply:
When we run `$ tox -e functional` it installs all the dependencies and such to `./.tox/functional/bin/...`. If tests were run with a differently named venv it could cause a problem if the base class is looking for `./.tox/functional/bin/...` and stuff is installed to say, `./.tox/some_new_env/bin/...`